.pip-2033.txt

```diff
@@ -0,0 +1,4 @@
+# Pip 20.3.4 no longer accepts wheels with pluses in their names,
+# and flit has not been updated to accomodate that quirk.
```

Comment:
```suggestion
# Pip 20.3.4 expects the wheel's metadata name to match the filename, 
# but flit mangles the build part of the version spec
```

Isn't the issue that `flit` isn't generating a wheel with the exact same name as the package,and pip has started checking for that?

That is, `pip` is actually expecting a `+` in the wheel name, not "no longer accepts wheels with pluses".

Reply:
pypa/pip#9628